go/ct/gen/state.go

```diff
@@ -227,6 +229,16 @@ func (g *StateGenerator) BindIsStorageCold(key Variable) {
 	g.storageGen.BindCold(key)
 }
 
+// BindIsTransientSet wraps TransientGenerator.BindSet.
+func (g *StateGenerator) BindIsTransientSet(key Variable) {
```

Comment:
This function name has some grammar issues ... how would one read a line of code like this:
```
var := Variable("a")
gen.BindIsTransientSet(var)
```
This is not very intuitive. How about something like
```
var := Variable("a")
gen.RestrictToStorageLocationWithNonZeroTransientStorage(var)
```
Granted, it is not the shortest name (maybe you have a better idea), but I would argue that when one is reading this, it is clearer what this line is supposed to do.

In general, I think the naming in the generator package needs a substantial overhaul ...